Tests/StencilTests/FilterSpec.swift

```diff
@@ -35,7 +35,23 @@ func testFilter() {
       }
 
       let result = try template.render(Context(dictionary: context, environment: Environment(extensions: [repeatExtension])))
-      try expect(result) == "Kyle Kyle with args value"
+      try expect(result) == "Kyle Kyle with args value1, \"value2\""
+    }
+    
+    $0.it("allows you to register a custom filter which accepts several arguments") {
+        let template = Template(templateString: "{{ name|repeat:'value\"1\"',\"value'2'\",'(key, value)' }}")
+        
```

Comment:
Theres lots of trailing whitespace here, could you please remove / configure editor not to put these here?